documentation/modules/snip-migrating-luns.adoc

```diff
@@ -0,0 +1,10 @@
+:_content-type: SNIPPET
+
+[NOTE]
+====
+* Unlike disk images that _are copied_ from a source provider to a target provider, LUNs are _detached_, _but not removed_, from virtual machines in the source provider and then attached to the virtual machines (VMs) that are created in the target provider.
+
+* LUNs are not removed from the source provider during the migration in case fallback to the source provider is required. However, before re-attaching the LUNs to VMs in the source provider, be sure that the LUNs are not used by VMs on the target environment at the same time, which might lead to data corruption.
```

Comment:
```suggestion
* LUNs are not removed from the source provider during the migration, in case fallback to the source provider is required. However, before re-attaching the LUNs to VMs in the source provider, verify that the LUNs are not used by VMs on the target environment at the same time, which might lead to data corruption.
```

Reply:
Good catch.